controllers/model/storage/local_storage_provider.go

```diff
@@ -0,0 +1,53 @@
+package storage
+
+import (
+	modelv1alpha1 "github.com/alibaba/kubedl/apis/model/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type LocalStorageProvider struct {
+}
+
+func (ls *LocalStorageProvider) CreatePersistentVolume(storage *modelv1alpha1.Storage, pvName string) *corev1.PersistentVolume {
+	pv := &corev1.PersistentVolume{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: pvName,
+		},
+
+		Spec: corev1.PersistentVolumeSpec{
+			AccessModes:                   []corev1.PersistentVolumeAccessMode{corev1.ReadWriteMany},
+			PersistentVolumeReclaimPolicy: corev1.PersistentVolumeReclaimRetain,
+			PersistentVolumeSource: corev1.PersistentVolumeSource{
+				Local: &corev1.LocalVolumeSource{
+					Path: storage.LocalStorage.Path,
+				},
+			},
+			Capacity: corev1.ResourceList{
+				corev1.ResourceStorage: resource.MustParse("50Mi"),
```

Comment:
50Mi may not be sufficient for all, shall we leave this setting to users through crd?

Reply:
+1

Reply:
actually, for hostpath volume, the capacity is not enforced.
This is here, because the api-server validation requires a value to be set.
I'll add a comment